mantidproject/mantid
# EnggGUI:: Vanadium curves now plotting for both banks after calibration

**Description of work.**

After calibration has completed on the Engineering Diffraction GUI, the vanadium curves should now be plotted for each bank separately. 

**To test:**

Go to: `Interfaces/Diffraction/Engineering Diffraction/Settings`
Go to the `Calibration` directory listed

Either:
- Delete the existing files which would only generate vanadium curves for one bank
- Change the calibration directory 

Files to delete should look something like this: `ENGINX_precalculated_vanadium_run00236516_bank_curves.nxs` & `ENGINX_precalculated_vanadium_run00236516_integration.nxs`
_(NOTE: Deleting these files or changing the calibration directory will force the GUI to run `EnggVanadiumCorrection` and produce the valid files)_

Go to: `Interfaces/Diffraction/Engineering Diffraction/Calibration`
Run Calibration with the default settings
Upon completion 

Check that there are now two plots produced for Vanadium Curves; one for each bank
Check that valid bank title given to each plot produced
Check that `Difc Zero` Peaks also still being plotted



Fixes #16151
_Does not need to be in the release notes._



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
